components/automate-ui/src/app/modules/desktop/desktop-detail/desktop-detail.component.html

```diff
@@ -18,20 +18,30 @@ <h2 class="heading-title">{{ desktop.name }}</h2>
     <div class="checkin-history-header">
       <h3>Check-in History</h3>
       <div class="heading-actions">
-        <chef-button secondary (click)="updateCheckInDays()">
-          <span>Last {{ (checkInNumDays === twoWeekNumDays ? 2 : 4) }} Weeks</span>
-          <chef-icon>expand_more</chef-icon>
-        </chef-button>
-        <chef-button secondary (click)="toggleDownloadDropdown()" class='download-dropdown-toggle'>
-          <span>Download</span>
-          <chef-icon>expand_more</chef-icon>
-        </chef-button>
-          <chef-dropdown class='download' [visible]="downloadDropdownVisible">
+        <div class="checkin-select">
+          <chef-button secondary (click)="toggleCheckInPeriodDropdown()" class='download-dropdown-toggle'>
+            <span>Last {{ (checkInNumDays === twoWeekNumDays ? 2 : 4) }} Weeks</span>
```

Comment:
we can shift this to the component.
`(checkInNumDays === twoWeekNumDays ? 2 : 4)`

Reply:
Yeah I thought so too at first when I looked at the code, but conditionally rendering the text here seems better than calling a function that wraps the same logic. So I let be untouched.